src/main/kotlin/calculator/StringCalculator.kt

```diff
@@ -0,0 +1,50 @@
+package calculator
+
+class StringCalculator {
+
+    fun calculate(input: String): Int {
+
+        if (input.isNullOrEmpty()) {
+            throw IllegalArgumentException()
+        }
+
+        var data = input.replace(" ", "")
+        verifyOperator(data)
+
+        val expression = Expression(data)
+        val queue = expression.apply()
+
+        var result: Int = Integer.parseInt(queue.poll())
+
+        if (queue.size % 2 != 0) {
+            throw IllegalStateException()
+        }
+
+        while (!queue.isEmpty()) {
```

Comment:
```suggestion
        while (queue.isNotEmpty()) {
```
코틀린은 정말 많은 기본 [확장 함수]를 제공합니다. 이를 하나씩 알아가보는 것도 좋은 학습이 될 것 같네요 :)

Reply:
넵!